### ISSUE TITLE
Info on python 3.8 `dumps` AttributeError

### ISSUE THREAD

**Issue by tlambert03** [napari/napari#1038]
## 🐛 Bug
Very low priority, and not really a napari issue, but just wanted to put this info somewhere.  We've seen an `AttributeError: 'NoneType' object has no attribute 'dumps'` in our tests on python 3.8.  It seems to have something to do with dask cleanup.  run the following _in a script_ (not in an interactive shell), in python 3.8:

```python
import napari
import dask.array as da

with napari.gui_qt():
    v = napari.view_image(da.ones((10, 10)))
```

when you quit, you see:
```
Exception ignored in: <function Pool.__del__ at 0x12b27eb80>
Traceback (most recent call last):
  File "/lib/python3.8/multiprocessing/pool.py", line 268, in __del__
  File "/lib/python3.8/multiprocessing/queues.py", line 362, in put
AttributeError: 'NoneType' object has no attribute 'dumps'
```

doesn't happen with a regular numpy array.  It _does_ seem to be a bug somehow in either dask or in the stdlib multiprocessing.  somehow the `multiprocessing.queues.SimpleQueue.put()` method is getting called as python is shutting down, when `multiprocessing.queues.context` has been cleaned up. and the `_ForkingPickler` object becomes none, causing the `AttributeError`.

**Comment by jni** [comment]
This appears to be the corresponding dask issue:

https://github.com/dask/dask/issues/5806

and it appears to have been fixed recently! 🎉 So we should test with the latest dask release (just this past weekend) and see if this is fixed.

**Comment by tlambert03** [comment]
Oh how did I miss that??  Thanks for linking

**Comment by jni** [comment]
@tlambert03 want to update your dask and give this another go? Maybe we can close this. =)

**Comment by tlambert03** [comment]
yeah fixed.  strangely enough, the earliest version I can find that fixes it was `2.11.0`, which was released on Feb 19th... a few weeks before this issue 😂
anyway. i'll close this now.  Not sure we need to pin any requirements, since it really doesn't affect anything but shutdown.  but for the record, it's fixed in dask 2.11.0 if anyone stumbles on this
